api/paths/paths.go

```diff
@@ -1,80 +1,138 @@
-package types
+package paths
 
 import (
 	"bytes"
 	"fmt"
 	"io"
+	"log"
```

Comment:
Why the move away from contextual logging in this package with `logrus`? And also why remove `gotil`? Just trying to reduce dependencies?